playlistmanager.lua

```diff
@@ -663,10 +661,63 @@ function draw_playlist()
 	
   ass:append(settings.style_ass_tags)
 
-  -- TODO: padding should work even on different osd alignments
-  if mp.get_property("osd-align-x") == "left" and mp.get_property("osd-align-y") == "top" then
-    ass:pos(settings.text_padding_x, settings.text_padding_y)
+  -- align from mpv.conf
+  local align_x = mp.get_property("osd-align-x")
+  local align_y = mp.get_property("osd-align-y")
+  -- align from style_ass_tags
+  if settings.style_ass_tags ~= nil then
+    local an_tag = settings.style_ass_tags:match('\\an%d')
+    if an_tag ~= nil then
+      local an = tonumber(an_tag:match('%d'))
+      if an == 1 then
+        align_x = 'left'
+        align_y = 'bottom'
+      elseif an == 2 then
```

Comment:
Maybe we can use a lookup table defined somewhere in the top of the file

```lua
local alignment_table = {
    [1] = { ["x"] = "left",   ["y"] = "bottom" },
    [2] = { ["x"] = "center", ["y"] = "bottom" },
    [3] = { ["x"] = "right",  ["y"] = "bottom" },
    [4] = { ["x"] = "left",   ["y"] = "middle" },
    [5] = { ["x"] = "center", ["y"] = "middle" },
    [6] = { ["x"] = "right",  ["y"] = "middle" },
    [7] = { ["x"] = "left",   ["y"] = "top" },
    [8] = { ["x"] = "center", ["y"] = "top" },
    [9] = { ["x"] = "right",  ["y"] = "top" },
}


function drawplaylist()
  local x = "\\an2"
  local an = tonumber(settings.style_ass_tags:match('\\an(%d)'))
  
  if an ~= nil then
    align_x = alignment_table[m]["x"]
    align_y = alignment_table[m]["y"]
  end
  ...
```

Reply:
`--osd-align-y=<top|center|bottom>`, so it's better to replaced `middle` by `center` in alignment_table